Install instructions via notebook for NCAR Jupyterhub users

ThomasMGeo opened the pull request:
Creating a notebook to go through the pytpe (package) install for new NCAR computing resources users. This also might be helpful for other jupyterhub users on non-NCAR resources.

ThomasMGeo committed:
Install notebook for NCAR users

Creating a notebook to go through the pytpe (package) install for new NCAR computing resources users. This also might be helpful for other jupyterhub users.

jsschreck commented:
Merge at will!

jsschreck commented:
Gabrielle also updated the evidential repo with instructions on getting the keras business running on the GPUs.

ThomasMGeo committed:
updating the sciparcs notebook

This notebook had an error previously for plotting confusion matricies. This was cleaned up (see old PR) along with more documentation.

ThomasMGeo commented:
Have updated the sciparcs intro notebook as well. Ready to merge whenever.
